### ISSUE TITLE
expand-button missing aria label

### ISSUE THREAD

**Issue by TimelyKimchiUser6** [MurhafSousli/ngx-sharebuttons#546]
#### I am submitting a

- [x] Bug Report

#### What is the expected behavior?
For accessibility, the <share-button> should have the aria-label set for expand button "...".

share-buttons > div > expand-button > button

#### What is the current behavior?
No name or aria label.
Consider "Expand Share Buttons"
